dashboard/lti_new.py

```diff
@@ -35,6 +38,22 @@
 COURSE_MEMBERSHIP = 'http://purl.imsglobal.org/vocab/lis/v2/membership'
 DUMMY_CACHE = 'DummyCache'
 
+# get CANVAS_DATA_ID_INCREMENT setting from env file
+env_json: Union[str, None] = os.getenv('ENV_JSON')
+if env_json:
+    # optionally load settings from an environment variable
+    ENV = hjson.loads(env_json)
+else:
+    # else try loading settings from the json config file
+    try:
+        with open(os.getenv("ENV_FILE", "/secrets/env.hjson")) as f:
+            ENV = hjson.load(f)
+    except FileNotFoundError as fnfe:
+        print("Default config file or one defined in environment variable ENV_FILE not found. This is normal for the build, should define for operation")
+        # Set ENV so collectstatic will still run in the build
+        ENV = os.environ
+CANVAS_DATA_ID_INCREMENT = ENV.get("CANVAS_DATA_ID_INCREMENT", 17700000000000000)
```

Comment:
I think we would want to import this from settings instead. I.e. `from django.conf import settings` then `settings.CANVAS_DATA_ID_INCREMENT`.

Reply:
Then you don't have to double process the config file.

Reply:
Great! Code updated.